src/package.cjs

```diff
@@ -9,11 +9,4 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-
-const vdom = require('./mdast-to-vdom');
-const types = require('./match-section-types');
-
-module.exports = {
-  vdom,
-  types,
-};
+module.exports = require('../package.json');
```

Comment:
Maybe it would be cleaner to have this as a pre-release step that just converts the package.json into a package.js module that can be imported properly.

Reply:
IDK, this is perfectly fine IMO :-)